Need support marked@"^6.0.0"

```
npm i marked-katex-extension
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: crx-ai-paper@0.0.0
npm ERR! Found: marked@6.0.0
npm ERR! node_modules/marked
npm ERR!   marked@"^6.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer marked@"^4 || ^5" from marked-katex-extension@3.0.0
npm ERR! node_modules/marked-katex-extension
npm ERR!   marked-katex-extension@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/why/.npm/_logs/2023-08-02T11_52_45_953Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/why/.npm/_logs/2023-08-02T11_52_45_953Z-debug-0.log

```

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
   },
   "homepage": "https://github.com/UziTech/marked-katex-extension#readme",
   "peerDependencies": {
-    "marked": "^4 || ^5"
+    "marked": "^4 || ^5 || ^6"
   },
   "devDependencies": {
     "@babel/core": "^7.22.9",
@@ -53,7 +53,6 @@
     "@semantic-release/github": "^9.0.4",
     "@semantic-release/npm": "^10.0.4",
     "@semantic-release/release-notes-generator": "^11.0.4",
-    "@types/marked": "^5.0.1",
     "babel-jest": "^29.6.2",
     "cheerio": "^1.0.0-rc.12",
     "eslint": "^8.45.0",
@@ -62,7 +61,7 @@
     "eslint-plugin-n": "^16.0.1",
     "eslint-plugin-promise": "^6.1.1",
     "jest-cli": "^29.6.2",
-    "marked": "^5.1.2",
+    "marked": "^6.0.0",
     "node-fetch": "^3.3.2",
     "rollup": "^3.27.0",
     "semantic-release": "^21.0.7",

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,1 +1,1 @@
-export default function markedKatex(options?: import('katex').KatexOptions): import('marked').marked.MarkedExtension
+export default function markedKatex(options?: import('katex').KatexOptions): import('marked').MarkedExtension

